tasks/build-doc.js

```diff
@@ -555,7 +555,7 @@ gulp.task('build-apps', ['clean-dist','less'], function () {
         'why-apps-over-connectors.md': 'Why should you choose Apps?',
         'using-oauth2.md': 'Using OAuth 2.0 to connect an App',
         'create-app-in-php.md': 'Create an App in PHP',
-        'access-scopes.md': 'Access scopes'
```

Comment:
I think there is a small error here. You should just add the "how to test my app" without removing the access scopes :)

Reply:
It's not a mistake, she moved the access scopes to the oauth2 page.

Reply:
(And I think it should stay like it was, we had a dedicated page on purpose.

Reply:
I merge the using oauth2 with access scope because it was a bit weird to have access scopes alone at the end of the navigation menu. And, how do they use Oauth without knowing the scopes?

Reply:
There is a link on the OAuth page to this scopes.

You have to consider the Access Scopes page as a reference page, when you are searching a scope to enable, you don't want to re-read the OAuth page, and it's also not obvious that it's hidden inside that page.

Reply:
I merge this PR and make another PR to get out the access scopes

Reply:
plz don't, if you merge, it will be automatically deployed.